app/models/linkset.rb

```diff
@@ -11,11 +11,21 @@ class Linkset < ApplicationRecord
       message: "does not appear to be a valid URL"
   end
 
+  after_commit :verify_linkbacks
+
   def empty?
     LINKS.map { |link| attributes[link] }.all?(&:blank?)
   end
 
   def update_attributes_from_gem_specification!(spec)
     update!(home: spec.homepage)
   end
+
+  def verified?(link)
```

Comment:
It looks like we need to add a test for this method to get the coverage percentage up to where it needs to be. At least it's an easy one! 🤗